teuthology-docs/config/definitions/teuthology-docs.yml

```diff
@@ -34,3 +34,12 @@
     builders:
       - shell:
           !include-raw ../../build/build
+
+    # publish docs to docs.ceph.com/teuthology/docs
+    publishers:
+      - ssh:
+          site: 'docs.ceph.com'
+          target: 'teuthology/docs/'
+          source: '.tox/docs/tmp/html/**'
+          remove-prefix: '.tox/docs/tmp/html/'
+          clean-remote: true
```

Comment:
Yeah, I'd added `clean-remote` to match the `--delete` command that we were passing to rsync, but it really does not behave the same way.

I'll just remove `clean-remote` for now.

A more robust solution would be to copy all the files to an individual time-stamped directory, and then exec a command that quickly moves the old directory out of the way and moves the new one into the old one's place. Or maybe use https://wiki.jenkins-ci.org/display/JENKINS/SSH+Agent+Plugin